0-ad-history-mod-package/simulation/templates/template_unit_champion_elephant.xml

```diff
@@ -18,6 +18,15 @@
   <Identity>
     <GenericName>War Elephant</GenericName>
     <VisibleClasses datatype="tokens">Elephant</VisibleClasses>
+    <History>
+        War elephants were incredibly unique and dangerous units. With their immense size, weight, and momentum, a couple of charging elephants could shatter entire formations and thus decide the outcome of a battle. Elephants were even able to damage or take down defensive fortifications. No wonder they are today often regarded as the “tanks of antiquity”.
+        Oftentimes, elephants also carried wooden platforms or turrets, with archers providing additional firepower. And just the presence of elephants alone rendered cavalry units virtually useless, as they scared off the horses.
+        But their psycholocal effect should not be underestimated either; the intimidating site alone of a charging and loudly trumpeting horde of elephants could spark heavy panic among enemies.
+        On the battlefield, elephants, in a constant fear of death, sometimes got out of control, turned back, and attacked their own line. To prevent that, riders had a special chisel to kill the elephant in the worst case. Elephants nonetheless, in a few instances, caused more damage to their own men than to the enemy's.
```

Comment:
```suggestion
        On the battlefield, elephants would sometimes get out of control and attack their own allies. To prevent that, riders had a special chisel to kill the elephant. Nonetheless, elephants sometimes caused more damage to their own men than to the enemy's.
```